### DISCUSSION TITLE
Minus Sign Displayed as Question Mark

### DISCUSSION THREAD

**Discussion by agungizzulhaq** [KaTeX/KaTeX#2473] [Q&A]
I use mysql to store the equation rendered by KaTeX. I've checked mysql table, and the minus sign displayed as it is. But, when i display it on the browser, it is changed to question mark.

Maybe, it is related to character encoding. But, i don't know how to fix it. Any ideas?

**Comment by ylemkimon** [comment]
Could you check the character code of the minus sign?
